draft-ietf-httpbis-alias-proxy-status.md

```diff
@@ -112,6 +112,17 @@ in which this parameter is not present.
 The proxy MAY send the empty string ("") as the value of `next-hop-aliases` to indicate that
 no CNAME records were encountered when resolving the next hop's name.
 
+# Implementation Considerations
+
+In order to include the `next-hop-aliases` parameter, a proxy needs to have access to the chain
+of alias names and canonical names received in CNAME records.
+
+Implementations ought to note that the full chain of names might not available in common DNS
+resolution APIs, such as `getaddrinfo`. `getaddrinfo` does have an option for `AI_CANONNAME`,
+but this will only return the last name in the chain (the canonical name), not the alias
+names. Implementations without access to the full chain MAY include incomplete information
+(such as only the canonical name).
```

Comment:
Is this the only circumstance under which the incomplete chain might be sent? In theory, there might be other circumstances where truncating the list might be useful. Maybe we could split off this `MAY` as a general allowance ?

Reply:
@LPardue I made the MAY more general — how does it look now?